ui/admin/tests/acceptance/targets/enable-session-recording/index-test.js

```diff
@@ -12,215 +12,218 @@ import { authenticateSession } from 'ember-simple-auth/test-support';
 import { TYPE_TARGET_SSH } from 'api/models/target';
 import select from '@ember/test-helpers/dom/select';
 
-module('Acceptance | targets | enable session recording', function (hooks) {
-  setupApplicationTest(hooks);
-  setupMirage(hooks);
-
-  let featuresService;
-  let storageBucketOne;
-  let storageBucketTwo;
-
-  const SETTINGS_LINK_SELECTOR = '.target-sidebar .hds-link-standalone';
-  const ENABLE_BUTTON_SELECTOR = '.target-sidebar .hds-button';
-  const DROPDOWN_SELECTOR = '[name=storage_bucket_id]';
-  const LINK_LIST_SELECTOR = '.link-list-item > a';
-  const LINK_LIST_SELECTOR_ITEM_TEXT = '.link-list-item__text';
-  const TOGGLE_SELECTOR = '[name=target-enable-session-recording]';
-  const LINK_TO_NEW_STORAGE_BUCKET =
-    '.enable-session-recording-toggle .hds-link-standalone';
-  const SAVE_BTN_SELECTOR = '.rose-form-actions [type="submit"]';
-  const CANCEL_BTN_SELECTOR = '.rose-form-actions [type="button"]';
-
-  // Instances
-  const instances = {
-    scopes: {
-      global: null,
-      org: null,
-    },
-    sessionRecording: null,
-  };
-
-  //URLs
-  const urls = {
-    globalScope: null,
-    orgScope: null,
-    projectScope: null,
-    targets: null,
-    target: null,
-    enableSessionRecording: null,
-    storageBuckets: null,
-    storageBucket: null,
-    newStorageBucket: null,
-  };
-
-  hooks.beforeEach(function () {
-    featuresService = this.owner.lookup('service:features');
-    // Generate resources
-    instances.scopes.global = this.server.create('scope', { id: 'global' });
-    instances.scopes.org = this.server.create('scope', {
-      type: 'org',
-      scope: { id: 'global', type: 'global' },
+module(
+  'Acceptance | targets | enable session recording | index',
```

Comment:
I only updated the module name here, as there is another file with the same module name